graphein/ppi/edges.py

```diff
@@ -0,0 +1,74 @@
+# %%
+# Graphein
+# Author: Arian Jamasb <arian@jamasb.io>, Ramon Vinas
+# License: MIT
+# Project Website: https://github.com/a-r-j/graphein
+# Code Repository: https://github.com/a-r-j/graphein
+import logging
+
+import networkx as nx
+import pandas as pd
+
+from graphein.ppi.parse_biogrid import BIOGRID_df
+from graphein.ppi.parse_stringdb import STRING_df
+
+log = logging.getLogger(__name__)
+
+
+def add_string_edges(G: nx.Graph, kwargs) -> nx.Graph:
```

Comment:
Did you mean for this to be the following?

```suggestion
def add_string_edges(G: nx.Graph, **kwargs) -> nx.Graph:
```